### ISSUE TITLE
网址被微信停止访问了

### ISSUE THREAD

**Issue by linyuxuanlin** [cnvoice/cnvoice#2]

**Comment by cnvoice** [comment]
Yes, the default url domain `heroku` has been banned in WeChat. One workaround is to customize your domain. Here is the reference: https://devcenter.heroku.com/articles/custom-domains
